config/dataset/test_ci_segmentation_binary.yaml

```diff
@@ -9,7 +9,7 @@ dataset:
   download_data: False
 
   # imagery
-  bands: [R, G, B]
+  bands: [1,2,3]
```

Comment:
maybe add the meaning of 1,2,3  which is which ?

Reply:
It's mentionned in the doc [here](https://github.com/NRCan/geo-deep-learning/tree/develop/dataset#single-band-vs-multi-band-imagery) already. Is that enough?